specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/ManagedEnvironments.json

```diff
@@ -0,0 +1,1928 @@
+          "$ref": "#/definitions/DestinationsConfiguration"
+        },
+        "tracesConfiguration": {
+          "description": "Open telemetry trace configuration",
+          "$ref": "#/definitions/TracesConfiguration"
+        },
+        "logsConfiguration": {
+          "description": "Open telemetry logs configuration",
+          "$ref": "#/definitions/LogsConfiguration"
+        },
+        "metricsConfiguration": {
+          "description": "Open telemetry metrics configuration",
+          "$ref": "#/definitions/MetricsConfiguration"
+        }
+      }
+    },
+    "DestinationsConfiguration": {
+      "description": "Configuration of Open Telemetry destinations",
+      "type": "object",
+      "properties": {
+        "dataDogConfiguration": {
+          "description": "Open telemetry datadog destination configuration",
+          "$ref": "#/definitions/DataDogConfiguration"
+        },
+        "otlpConfigurations": {
+          "description": "Open telemetry otlp configurations",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OtlpConfiguration"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "DataDogConfiguration": {
+      "description": "Configuration of datadog ",
+      "type": "object",
+      "properties": {
+        "site": {
+          "description": "The data dog site",
+          "type": "string"
+        },
+        "key": {
+          "description": "The data dog api key",
+          "type": "string",
+          "x-ms-secret": true
+        }
+      }
+    },
+    "OtlpConfiguration": {
+      "description": "Configuration of otlp ",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of otlp configuration",
+          "type": "string"
+        },
+        "endpoint": {
+          "description": "The endpoint of otlp configuration",
+          "type": "string"
+        },
+        "insecure": {
+          "description": "Boolean indicating if otlp configuration is insecure",
+          "type": "boolean"
+        },
+        "headers": {
+          "description": "Headers of otlp configurations",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Header"
+          },
+          "x-ms-identifiers": [
+            "key"
+          ]
+        }
+      }
+    },
+    "Header": {
+      "description": "Header of otlp configuration",
+      "type": "object",
+      "properties": {
+        "key": {
+          "description": "The key of otlp configuration header",
+          "type": "string"
+        },
+        "value": {
+          "description": "The value of otlp configuration header",
+          "type": "string"
+        }
+      }
+    },
+    "TracesConfiguration": {
+      "description": "Configuration of Open Telemetry traces",
+      "type": "object",
+      "properties": {
+        "includeDapr": {
```

Comment:
> "includeDapr": { [](http://example.com/codeflow?start=7&length=17)

consider fixing the warnings

[https://github.com/Azure/azure-rest-api-specs/pull/30562/checks?check_run_id=30773553891](https://github.com/Azure/azure-rest-api-specs/pull/30562/checks?check_run_id=30773553891)